helpers/content.go

```diff
@@ -546,9 +546,19 @@ func getAsciidocContent(content []byte) []byte {
 	jww.INFO.Println("Rendering with", path, "...")
 	cmd := exec.Command(path, "--no-header-footer", "--safe", "-")
 	cmd.Stdin = bytes.NewReader(cleanContent)
-	var out bytes.Buffer
+	var out, cmderr bytes.Buffer
 	cmd.Stdout = &out
-	if err := cmd.Run(); err != nil {
+	cmd.Stderr = &cmderr
+	err := cmd.Run()
+	// asciidoctor has exit code 0 even if there are errors in stderr
+	// -> log stderr output regardless of state of err
+	for _, item := range strings.Split(string(cmderr.Bytes()), "\n") {
+		item := strings.TrimSpace(item)
+		if item != "" {
+			jww.ERROR.Println(item)
```

Comment:
Yes, I can do that. Putting `page.Path` in `DocumentName` we can get error messages like
```
ERROR: 2016/10/13 09:47:12 content.go:560: asciidoctor: ERROR: post/fail.ad: line 139: mismatched macro: endif::backen-docbook[], expected endif::backend-docbook[]
```
I'll add a second commit to this pull request with the changes for you to review. Ok?